codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -1181,6 +1181,10 @@ public void deactivate(final DefaultStream stream, final Iterator<?> itr) {
                 pendingEvents.add(new Event() {
                     @Override
                     public void process() {
+                        // If the stream has already been closed, then don't bother to try and close again.
+                        if (stream.parent() == null) {
```

Comment:
Oops, sorry for the late response ... must have gotten lost in my inbox.  Yeah we can ship it.

@Scottmitch could you just expand the comment to make it more clear?